requirements.txt

```diff
@@ -5,6 +5,7 @@ Markdown==3.3.7
 bencode.py==4.0.0
 django-countries==7.3.2
 django-extensions==3.1.3
+django-simple-captcha==0.5.17
```

Comment:
Not sure if I like the captch dependency, we don't have it for flagging packages. Do you think it's really required?

Reply:
Well this is what people said I should put in when I was asking in the arch channels, I don't mind dropping it but to reduce spam this is what was suggested.

To quote: *"Just make a captcha thing"*